docs/aca/10-aca-iac-bicep/index.md

```diff
@@ -273,47 +278,173 @@ To achieve this, add a new file under the `bicep` directory as shown below:
 Start by creating a new resource group which will contain all the resources to be created by the Bicep scripts.
 
 ```Powershell
-az group create `
---name "<your RG name>" `
---location "eastus"  
-```
-
-Next create an ACR inside the newly created Resource Group:
+$RESOURCE_GROUP="<your RG name>"
+$LOCATION="<your location>"
 
-```Powershell
-az acr create `
-    --resource-group <your RG name> `
-    --name <your ACR name>`
-    --sku Basic `
-    --admin-enabled true
+az group create `
+--name $RESOURCE_GROUP `
+--location $LOCATION
 ```
 
-!!! note
-    Once the RG and ACR are created you can check [this section](../../aca/08-aca-monitoring/index.md#2-build-new-images-and-push-them-to-acr) which shows the different commands to build and push the images to ACR. Make sure you are at the root project directory when executing the aforementioned commands. Finally run the bicep file against the newly created Resource Group as shown below.
-
-With the steps above completed we are ready to deploy all the different resources. We just need to create a parameters file which will simplify the invocation of the main bicep file. 
+Create a parameters file which will simplify the invocation of the main bicep file. To achieve this, right click on file `main.bicep` and select **Generate Parameter File**. 
+This will result in creating a file named `main.parameters.json` similar to the file below:
 
-To achieve this, right click on file `main.bicep` and select **Generate Parameter File**. This will result in creating a file named `main.parameters.json` similar to the file below:
+??? example
 
-???+ example
     === "main.parameters.json"
     
         ```json
         --8<-- "https://raw.githubusercontent.com/Azure/aca-dotnet-workshop/main/bicep/main.parameters.json"
         ```
-
 !!! note
-    
+
     To use this file, you need to edit this generated file and provide values for the parameters. You can use the same values shown above in sample file. 
-    
-    You only need to replace parameter values between the angle brackets `<>` with values related to your ACR resource and SendGrid.
+
+    You only need to replace parameter values between the angle brackets `<>` with values related to your container registry and SendGrid. Values for container registry and container images can be dervied by following
+    one of the three options in next step.
+
+Next, we will prepare container images for the three container apps and update the values in `main.parameters.json` file. You can do so by any of the three options below:
+
+=== "Option 1: Build and Push the Images to Azure Container Registry (ACR)"
+
+    1. Create an Azure Container Registry (ACR) inside the newly created Resource Group:
+
+        ```Powershell
+        $CONTAINER_REGISTRY_NAME="<your ACR name>"
+
+        az acr create `
+            --resource-group $RESOURCE_GROUP `
+            --name $CONTAINER_REGISTRY_NAME `
+            --sku Basic `
+            --admin-enabled true
```

Comment:
@pankajagrawal16 Why is admin-enabled set to true if you are using managed identity?

Reply:
Good point. We can probably remove admin enabled now. I will fix it.